app/assets/stylesheets/components/_event.sass

```diff
@@ -0,0 +1,24 @@
+$Event-backgroundColor: Theme-color(background, default)
+$Event-borderTop-color: #FF623F  // FIXME Put this color on background color MAP on theme.sass file.
```

Comment:
I think we can go with the second one yes. It makes more sense to me at the moment :)